.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
 jobs:
   include:
     - stage: test
+      script: ./csu ci load_content
       script: ./csu ci test_general
```

Comment:
Would this be better in a distinct test from `load_content`?

Reply:
`load_content` is the test that I've added, and I believe it is run as a distinct test in the same stage (`test`). Correct me if I'm wrong.

Reply:
This will run `load_content` with `test_general`, just change line 19 to be:
```
    - script: ./csu ci test_general
```
Much like the lines following it. Each script in the same stage is a new item of a different key. Very un-intuitive YAML file!

Reply:
Ohhhh you're talking about the missing `-`, with you now. That was just a typo haha